### ISSUE TITLE
Mac Catalyst build not working on 0.13.x & 0.14.x versions

### ISSUE THREAD

**Issue by rogerkerse** [walterholohan/react-native-crisp-chat-sdk#136]
So when I update this package from 0.12.x to anything above it (0.13.x or 0.14.x for now) then build for Mac Catalyst platform starts to fail. Couldn't find any concrete visible error.

So right now it is quite annoying to release the app for iOS and Mac Catalyst.
1. For iOS 0.12.x does not work. It gives `Asset validation failed (90482) Invalid Executable. The executable 'app.app/Frameworks/Crisp.framework/Crisp' contains bitcode.` error. so that I have to update to `0.14.x`.
2. Mac build requires `0.12.x` to work.

Basically to build for both platforms I have to install different dependencies.
